overlord/snapstate/snapmgr.go

```diff
@@ -373,8 +372,6 @@ func (m *SnapManager) doUnlinkSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	// mark as inactive
-	snapst.Active = false
```

Comment:
just removing this seems dangerous,  we have removed current symlink and wrappers, it seems a bit of lie to say we have a Current